package.json

```diff
@@ -398,17 +422,24 @@
   },
   "extensionDependencies": [
     "vscode.docker",
-    "vscode.yaml"
+    "vscode.yaml",
+    "ms-vscode.azure-account"
   ],
   "devDependencies": {
     "vscode": "^1.0.0",
     "typescript": "^2.1.5",
-    "@types/node": "^6.0.40"
+    "@types/node": "^6.0.40",
+    "@types/keytar": "^4.0.1"
   },
   "dependencies": {
     "dockerfile-language-server-nodejs": "^0.0.7",
     "dockerode": "^2.5.1",
     "vscode-extension-telemetry": "^0.0.6",
-    "vscode-languageclient": "^3.1.0"
+    "vscode-languageclient": "^3.1.0",
+    "azure-arm-containerregistry": "^1.0.0-preview",
+    "azure-arm-resource": "^2.0.0-preview",
+    "azure-arm-website": "^1.0.0-preview",
+    "docker-hub-api": "^0.5.1",
```

Comment:
docker-hub-api is licensed under the [GPLv3](https://github.com/RyanTheAllmighty/Docker-Hub-API/blob/b00c8adf6bc1184a997263354762ea37533e2af1/LICENSE). I'm not a lawyer but you all might want to tread carefully here.